### PR TITLE
fix: PushModal pushing page to PageStack

### PR DESCRIPTION
**Pull request by RLittlesII**
**What kind of change does this PR introduce?**

Fixes Modal Navigation.

**What is the current behavior?**

Resolves: #94 

Currently the code is pushing the page to the `NavigationView` when it shouldn't.

**What is the new behavior?**

The modal properly pushes with no error.

**What might this PR break?**

Modal Navigation

**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: Sample/SextantSample.Android/SextantSample.Android.csproj [hunk 1]
```diff
@@ -1,117 +1,117 @@
...
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="System.Reactive" Version="4.1.5" />
+        <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.3" />
+        <PackageReference Include="Xamarin.Android.Support.Design" Version="28.0.0.1" />
+        <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.1" />
+        <PackageReference Include="Xamarin.Android.Support.v4" Version="28.0.0.1" />
+        <PackageReference Include="Xamarin.Android.Support.v7.CardView" Version="28.0.0.1" />
+        <PackageReference Include="Xamarin.Android.Support.v7.MediaRouter" Version="28.0.0.1" />
+        <PackageReference Include="ReactiveUI.XamForms" Version="9.18.12" />
```
Comment (glennawatson) [inline Sample/SextantSample.Android/SextantSample.Android.csproj]: Should we go back to the 9.* format?

File: Sample/SextantSample.iOS/SextantSample.iOS.csproj [hunk 2]
```diff
@@ -1,178 +1,178 @@
...
+        </ImageAsset>
+    </ItemGroup>
+    <ItemGroup>
+        <Reference Include="System" />
+        <Reference Include="System.Xml" />
+        <Reference Include="System.Core" />
+        <Reference Include="Xamarin.iOS" />
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="ReactiveUI.XamForms" Version="9.18.12" />
```
Comment (glennawatson) [inline Sample/SextantSample.iOS/SextantSample.iOS.csproj]: Same question for here?

Comment (RLittlesII) [reply Sample/SextantSample.iOS/SextantSample.iOS.csproj]: Yes we should.  This is temporary.

File: Sample/SextantSample.Android/SextantSample.Android.csproj (hunk 1 above)
Comment (RLittlesII) [reply Sample/SextantSample.Android/SextantSample.Android.csproj]: Yes we should.  This is temporary.

**Pull request merged**